File: 1561. Maximum Number of Coins You Can Get/Maximum Number of Coins You Can Get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        piles.sort()
        result = 0
        i = len(piles)//3
        while i < len(piles):
            result = result + piles[i]
            i = i + 2
        return result

File: 1561. Maximum Number of Coins You Can Get/Maximum Number of Coins You Can Get.cpp
class Solution {
public:
    int maxCoins(vector<int>& piles) {
        sort(piles.begin(),piles.end());
        int result = 0;
        int i = piles.size() / 3;
        while(i < piles.size()){
            result = result + piles[i];
            i = i + 2;
        }
        return result;
    }
};
